exec/java-exec/src/main/codegen/data/MathFunc.tdd

```diff
@@ -138,7 +138,7 @@ unaryMathFunctions : [
       {input: "UInt8", outputType: "UInt8", castType: "long"}
      ] 
     },
-    {className: "LeftShift", funcName: "lshift", javaFunc: " << ", types: [
+    {className: "LeftShift", funcName: "lshift", aliasName: "", javaFunc: " << ", types: [
```

Comment:
Would we want this for all `INT` types like `SHORT`, `TINYINT` etc?

Reply:
See my comment for `div`